README.md

```diff
@@ -1,2 +1,63 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This is a plugin for Errbot (http://errbot.io) to manage a "topic-a-day" channel for daily discussion of topics
+
+## Installation
+To install Errbot, please use the manual [here](https://errbot.readthedocs.io/en/latest/user_guide/setup.html)
+
+## Usage
+TODO: FILL THIS
+
+## Validation
+When running the program, some validations will run and if they are not configured properly an exception will be raised with the appropriate response
+
+# Slack commands
+These are the command you can send privately to the bot and he will respond appropriately
+
+## add_topic
+send it the topic and the topic name and it will add it to the back of the queue
+
+## list_topics
+shows all the topics in all of the states:
+- used (previous)
+- unused (upcoming)
+
+# Environment Variables
+If you don't know how to do it read at the bottom
+
+## Customizable
+In this section you can see all of the environment variables that can be used.
+
+### TOPIC_CHANNEL
+The name of the channel to post in
+This value is required
+the value should start with a pound key(`#`) as all channels do 
+
+
+### TOPIC_DAYS
+Days to post a topic. Comma separated list of day names
+the default is Monday to Friday 
+
+For example TOPIC_DAYS="monday,tuesday,wednesday,thursday,friday"
+
+### TOPIC_TIME
+What time the topic is posted every day, 24hr notation
+the default is 9AM
+
+### TOPIC_POLLER_INTERVAL
+How frequently the poller runs. Lower numbers might result in higher load
+the default is 5, and it's an integer value (so no funny business with 2.5 okay?)
+
+## Automatic
+These are generated from previous ones, they will show up in the config but don't touch them
+
+### TOPIC_CHANNEL_ID
+channel id that is extracted from the channel name
+
+## Add environment Variable linux
```

Comment:
This section is not necessary and can be removed. We do not need to tell users how to set environment variables as that will vary based on their installation. For example, these instructions would not work for my own errbot install or even on my local machine - I run zsh.